src/toast/__tests__/toast.test.js

```diff
@@ -0,0 +1,200 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+import {mount} from 'enzyme';
+import {Toast, StyledBody, StyledCloseIcon, KIND} from '../index';
+
+jest.useFakeTimers();
+
+describe('Toast', () => {
+  test('basic functionality', () => {
```

Comment:
tests names seems to be very generic and do not describe their meaning and "under-test" functionality. It's harder to maintain and fix tests without more descriptive naming.

Reply:
@blackswanny do you have any suggestions here?

Reply:
this seems to be `should render Toast with default settings` test.
I try to avoid usage of `toMatchSnapshot`, but it works well for initial render check and styled components check.
```js

  test.each([
    [TYPE.search, false],
    [TYPE.select, false],
    [TYPE.search, true],
    [TYPE.select, true],
  ])(
    'should render component in %s mode and %s for multiple choice',
    (type, multiple) => {
      allProps.type = type;
      allProps.multiple = multiple;
      wrapper = mount(<Select {...allProps} />);
      expect(wrapper).toMatchSnapshot(
        'Component has correct render in ' +
          type +
          ' mode and mutiple choice equal ' +
          multiple,
      );
    },
  );
```